backend/SC4SNMP_UI_backend/inventory/routes.py

```diff
@@ -11,6 +11,16 @@
 mongo_groups = mongo_client.sc4snmp.groups_ui
 mongo_inventory = mongo_client.sc4snmp.inventory_ui
 
+def get_inventory_type(document):
+    if not document["address"][0].isdigit():
```

Comment:
now I have some doubts about checking it like this, I think domain can start with number as well
Is it the same validationa s in core SC4SNMP?

Reply:
I must check how it works in core sc4snmp, but here if the address starts begins with digit it is automatically assigned as a host. It can be either IP or host name. The only restriction is that the group can't begin with a digit. This `if statement` was added only to detect the objects which are for sure not groups, so that we don't need to make an extra query to mongo.

However currently there is validation in the frontend that doesn't allow host names to start with a digit. I can change that